snooty/postprocess.py

```diff
@@ -648,7 +702,45 @@ def get_include_argument(node: n.Directive) -> str:
             assert include_page is not None
             ast = include_page.ast
             assert isinstance(ast, n.Parent)
-            node.children = [util.fast_deep_copy(ast)]
+
+            deep_copy_children: MutableSequence[n.Node] = [util.fast_deep_copy(ast)]
+
+            # TODO: Move subsgraphing implementation into parse layer, where we can
+            # ideally take subgraph of the raw RST
+            start_after_text = (
+                node.options["start-after"] if "start-after" in node.options else ""
```

Comment:
You can use Python dict's [`get()`](https://docs.python.org/3/library/stdtypes.html#dict.get) to safely attempt to retrieve this value. If it's not found, `None` will be returned, which also seems a bit cleaner to me than an empty string.
```suggestion
                node.options.get("start-after")
```